freestyle/shared/src/main/scala/freestyle/module.scala

```diff
@@ -98,6 +102,10 @@ object openUnion {
       q"""new {
         ..$parsed
       }"""
+
+    if (showTrees)
+      c.echo(c.enclosingPosition, s"generated tree:\n${showCode(expr)}")
+
```

Comment:
Aah-- I'm just getting my grounding so I didn't even look at the macro interplay! I can remove it from `openUnion`.